apps/fz_http/lib/fz_http/config.ex

```diff
@@ -25,9 +25,17 @@ defmodule FzHttp.Config do
     def fetch_env!(app, key) do
       application_env = Application.fetch_env!(app, key)
 
-      with :error <- fetch_process_value(key),
-           :error <- fetch_process_value(get_last_pid_from_pdict_list(:"$ancestors"), key),
-           :error <- fetch_process_value(get_last_pid_from_pdict_list(:"$callers"), key) do
+      with :error <- fetch_process_value(key_function(app, key)),
+           :error <-
+             fetch_process_value(
+               get_last_pid_from_pdict_list(:"$ancestors"),
+               key_function(app, key)
+             ),
+           :error <-
+             fetch_process_value(
+               get_last_pid_from_pdict_list(:"$callers"),
+               key_function(app, key)
+             ) do
```

Comment:
```suggestion
      pdict_key = key_function(app, key)
      with :error <- fetch_process_value(pdict_key),
           :error <- fetch_process_value(get_last_pid_from_pdict_list(:"$ancestors"), pdict_key),
           :error <- fetch_process_value(get_last_pid_from_pdict_list(:"$callers"), pdict_key) do
```

Reply:
^ this will prevent function calling over and over again and make formatted code nicer